project_task_category/models/project_task_category.py

```diff
@@ -11,4 +11,5 @@ class ProjectTask(models.Model):
     categ_id = fields.Many2one(
         'project.category',
         'Category',
+        domain=[('code', '=', 'categ_id')]
```

Comment:
@dreispt We only need one set of categories (that independent
if have or not parent_id or are assigned on the Task as root task category) 
are available just for the Task categ_id field.

And we think no need to have different sets for different Projects because this categories are very generic.

Don't you think so?